Parse/src/main/java/com/parse/ParseQuery.java

```diff
@@ -1052,9 +1038,7 @@ public CachePolicy getCachePolicy() {
    *
    * @see ParseQuery#setCachePolicy(CachePolicy)
    */
-  /* package */ ParseQuery<T> fromNetwork() {
-    checkIfRunning();
-
+  public ParseQuery<T> fromNetwork() {
```

Comment:
Yes! Before this being exposed, if you had a `ParseQuery` and called `query.fromPin()` or `query.fromLocalDatastore()` there was no way to switch back to network from the same query object. This PR allows that and also allows calling `find()` multiple times from the same object.